### DISCUSSION TITLE
Per species force RMSE

### DISCUSSION THREAD

**Discussion by omidshy** [mir-group/allegro#67] [Q&A] [answered]
Hi guys,

I'm training an Allegro (or NequIP) model for a system containing 7 different chemical species, and I consistently get rather higher RMSE on predicted forces for one of the species (F) (on both training and validation sets). I can not find anything that makes these F atoms very different from the other species, regarding the number of F atoms and the magnitude of their forces in the training set. I was wondering if this is somehow related to the per-species scale and shift parameters in the Allegro model.

**Comment by Linux-cpp-lisp** [comment, marked as answer]
Hi @omidshy ,

Some species may just be harder to describe and have more complex interactions than others.  You can try using a per-species normalized loss on the forces (similar to https://github.com/mir-group/nequip/blob/main/configs/full.yaml#L234-L237 but with an L2, for example) and see if this helps the network to learn all elements well despite any imbalance in the chemical composition.
